sechub-doc/src/docs/asciidoc/documents/pds/pds_config.adoc

```diff
@@ -79,18 +79,18 @@ workspace location can be defined
 
 [[section-pds-server-config-file]]
 === Server configuration file
-PDS reads configuration JSON file on startup.
+The {pds} reads a configuration JSON file on startup to configure the product delegation server 
+and the provided products.
 
-*Location*
+*Location* +
+Per default the file is located at `./pds-config.json`
```

Comment:
Relative pathes are imho not helpful here:
```suggestion
Per default the file is located at `/pds/pds-config.json`
```
If it's the directory from where the PDS .jar has been started, then a hint should be added that this directory is `/pds` in our container images.